rotkehlchen/tests/unit/globaldb/test_globaldb_consistency.py

```diff
@@ -86,6 +86,7 @@ def as_dict(self) -> dict[str, Any]:
         }
 
 
+@pytest.mark.skipif('CI' in os.environ, reason='skipped in CI until remote asset updates are fixed')  # noqa: E501
```

Comment:
make a note for that and link it here. What exactly do you need to fix? Will you fix it? If not then who will?

Reply:
https://github.com/orgs/rotki/projects/11/views/2?pane=issue&itemId=89977797